ClearKeep/Scenes/Login/Views/LoginContentView.swift

```diff
@@ -0,0 +1,225 @@
+
+							Spacer()
+
+							Button("Login.ForgotPassword?".localized) {}
+							.padding()
+							.font(fontSignIn)
+							.foregroundColor(foregroundColorViewBt)
+						}
+						VStack(spacing: Constant.spacerBottomView) {
+							// MARK: - Line between 2 views
+							Rectangle().frame(height: Constant.heightRectangle)
+								.padding(.horizontal).foregroundColor(foregroundColorWhite)
+
+							Text("Login.SignInWith".localized)
+								.font(fontSignIn)
+								.foregroundColor(foregroundColorWhite)
+
+							HStack(spacing: Constant.spacerBottom) {
+								Button(action: { }, label: {
+									Text("")
+									iconGoogle
+								})
+									.frame(width: Constant.widthIconBt, height: Constant.widthIconBt)
+
+								Button(action: { }, label: {
+									Text("")
+									iconOffice
+								})
+									.frame(width: Constant.widthIconBt, height: Constant.widthIconBt)
+
+								Button(action: { }, label: {
+									Text("")
+									iconFacebook
+								})
+									.frame(width: Constant.widthIconBt, height: Constant.widthIconBt)
+							}
+						}
+					}
+					.frame(width: Constant.widthSignInBt)
+
+					Spacer(minLength: Constant.spacerBottom)
+				}
+
+				// MARK: - Social login button
+				VStack {
+					// MARK: - Sign up button
+					Text("Login.Don'tHaveAnAccount?".localized)
+						.font(fontSignIn)
+						.foregroundColor(foregroundColorWhite)
+
+					Spacer(minLength: Constant.spacer)
+
+					Button("Login.SignUp".localized) {}
+					.frame(width: Constant.widthSignUpBt, height: Constant.heightBt)
+					.font(fontSignIn)
+					.foregroundColor(foregroundColorViewBt)
+					.overlay(
+						RoundedRectangle(cornerRadius: Constant.radius)
+							.stroke(foregroundColorViewBt, lineWidth: Constant.lineWidthBorder))
+
+					Spacer(minLength: Constant.spacerBottomView)
+
+					Text("Login.AppVersion:1.1.1".localized)
+						.font(AppTheme.shared.fontSet.font(style: .placeholder3))
+						.foregroundColor(foregroundColorWhite)
+				}
+
+			}
+			.background(backgroundColorView)
+			.edgesIgnoringSafeArea(.all)
+		}
+	}
+}
+
+// MARK: - Private func
+private extension LoginContentView {
+	var imageLogo: Image {
+		AppTheme.shared.imageSet.logo
+	}
+
+	var iconLock: Image {
+		AppTheme.shared.imageSet.lockIcon
+	}
+
+	var iconMail: Image {
+		AppTheme.shared.imageSet.mailIcon
+	}
+
+	var iconGoogle: Image {
+		AppTheme.shared.imageSet.googleIcon
+	}
+
+	var iconOffice: Image {
+		AppTheme.shared.imageSet.officeIcon
+	}
+
+	var iconFacebook: Image {
+		AppTheme.shared.imageSet.facebookIcon
+	}
+
+	var backgroundColorView: LinearGradient {
+		colorScheme == .light ? backgroundColorGradient : backgroundColorDark
+	}
+
+	var backgroundSignInBt: LinearGradient {
+		colorScheme == .light ? backgroundColorWhite : backgroundColorGradient
+	}
+
+	var foregroundColorView: Color {
+		colorScheme == .light ? foregroundColorPrimary : foregroundColorWhite
+	}
+
+	var foregroundColorViewBt: Color {
```

Comment:
@dongnh99 : Do not abbreviate!
```suggestion
	var foregroundColorViewButton: Color {
```